src/shared/components/app/app.tsx

```diff
@@ -54,6 +54,11 @@ export class App extends Component<AppProps, any> {
             >
               {I18NextService.i18n.t("jump_to_content", "Jump to content")}
             </button>
+            <link
+              rel="stylesheet"
+              href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/styles/atom-one-dark.min.css"
```

Comment:
Check out the file `src/shared/components/app/styles.scss` , that's where you include these locally.

Reply:
Instead made it handled like the site themes to support both light + dark since I couldnt figure out a way to do that in styles.scss